docker/dev/compose.dev.yml

```diff
@@ -12,37 +13,46 @@ services:
       - "80:80"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+    networks:
+      - frontend
+      - users
 
   client:
     build: 
       context: ${DTAAS_DIR}/
       dockerfile: ${DTAAS_DIR}/docker/dev/client.dockerfile
+    restart: unless-stopped
     volumes:
-      - "${DTAAS_DIR}/client/config/local.js:/dtaas/client/build/env.js"
```

Comment:
The developers use configs from the regular code base - not from `deploy/` directory. Please revert this change.